runtime/rococo/src/lib.rs

```diff
@@ -849,7 +849,12 @@ sp_api::impl_runtime_apis! {
 
 	impl authority_discovery_primitives::AuthorityDiscoveryApi<Block> for Runtime {
 		fn authorities() -> Vec<AuthorityDiscoveryId> {
-			AuthorityDiscovery::authorities()
+			let mut authorities = AuthorityDiscovery::authorities();
```

Comment:
we can have `get_historical_authority_ids` return keys also for the current session